lxd/networks.go

```diff
@@ -1854,6 +1854,21 @@ func (n *network) Start() error {
 
 		// Spawn DNS forwarder if needed (backgrounded to avoid deadlocks during cluster boot)
 		if dnsClustered {
+			// Create forkdns servers directory
+			if !shared.PathExists(shared.VarPath("networks", n.name, forkdnsServersListPath)) {
```

Comment:
Are we expecting another file for forkdns? If not, why not just have one file in the existing folder instead?

Reply:
Also I'm not seeing forkdnsServersListPath here or in earlier commits, did I miss something?

Reply:
@stgraber because I create a temporary file and then move it into place once its created, this creates a new file and the old watcher is lost. I took inspiration from https://github.com/lxc/lxd-demo-server/blob/b0b97616a167ce42019100eec818b1c9dd293cd2/main.go#L123-L133 in how to overcome this.

Reply:
@stgraber yes I went through a few iterations on this and things have got a bit muddled, will rework the commits.

Reply:
Ah, doh, forgot about inotify being a bit annoying sometimes, makes sense :)